File: benchmarks/interpreter/bench_strings.py
s = ""
for i in range(10000): s += "hello"
print(f"concat len: {len(s)}")
total = 0
for j in range(1000):
    parts = "a,b,c,d,e,f,g,h".split(",")
    total += len(parts)
print(f"split parts: {total}")

File: benchmarks/interpreter/bench_strings.pl
my $s = ""; for my $i (0..9999) { $s .= "hello"; }
print "concat len: " . length($s) . "\n";
my $total = 0; for my $j (0..999) { my @p = split(",", "a,b,c,d,e,f,g,h"); $total += scalar(@p); }
print "split parts: $total\n";
